pkg/gatewayserver/gatewayserver.go

```diff
@@ -210,6 +217,29 @@ func New(c *component.Component, conf *Config, opts ...Option) (gs *GatewayServe
 
 	hooks.RegisterUnaryHook("/ttn.lorawan.v3.NsGs", cluster.HookName, c.ClusterAuthUnaryHook())
 
+	for name, prefix := range gs.forward {
+		if name == "" {
+			gs.upstreamHandlers["cluster"] = ns.NewHandler(ctx, "cluster", c, prefix)
+		} else {
+			str := strings.SplitN(name, ":", 2)
+			if len(str) != 2 {
+				continue
```

Comment:
I think we should just error on invalid config, right?

Reply:
Still open

Reply:
Yeah ok. I guess that would make it consistent with other config handling.